src/js/game/LevelMVC/LevelView.js

```diff
@@ -794,14 +794,14 @@ module.exports = class LevelView {
     return tween;
   }
 
-  playPlaceBlockAnimation(position, facing, blockType, blockTypeAtPosition, completionHandler) {
+  playPlaceBlockAnimation(position, facing, blockType, blockTypeAtPosition, entity, completionHandler) {
     var jumpAnimName;
     let blockIndex = this.yToIndex(position[1]) + position[0];
 
-    if (blockType === "cropWheat" || blockType === "torch" || blockType.startsWith("rail") || blockType.startsWith("redstoneWire")) {
+    if (entity === this.agent || blockType === "cropWheat" || blockType === "torch" || blockType.startsWith("rail") || blockType.startsWith("redstoneWire")) {
```

Comment:
similarly, let's move this blocktype checking into LevelBlock

Reply:
👍

Reply:
So looking into this:
The playPlaceBlockAnimation doesn't -have- the block itself, just the blockType string. How do you want to proceed?
(1) keep it the way it is
(2) add a function to leveView that can evaluate blockType strings to determine whether or not the animation should play
(3) update every instance of this function call to take a block instead of a string (blockType) and then use a method in LevelBlock to do this check

Reply:
Of those three, I like (3) the best, but I'd also be fine with just adding a static method on to LevelBlock like we did with [`isMiniblock`](https://github.com/code-dot-org/craft/blob/master/src/js/game/LevelMVC/LevelBlock.js#L182)